src/Symfony/Component/Form/Tests/Extension/Core/Type/MoneyTypeTest.php

```diff
@@ -33,7 +33,7 @@ public function testPassMoneyPatternToView()
         $view = $this->factory->create(static::TESTED_TYPE)
             ->createView();
 
-        $this->assertSame('{{ widget }} €', $view->vars['money_pattern']);
+        $this->assertSame('{{ widget }} &euro;', $view->vars['money_pattern']);
```

Comment:
These test changes look suspicious to me

Reply:
yeah we need 2 tests for utf+iso. The entity is direct effect of htmlentities, and what this topic is about: euro sign not being available in ISO (hence we use entities).